components/coding/profileV2/BadgesDisplayedComponent.tsx

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from "@apollo/client";
+import { format } from "date-fns";
+import React, { useState } from "react";
+import { FETCH_TOTAL_USER_BADGES_COUNT } from "../../../graphql/fetchTotalUserBadgesCount";
+import {
+  FetchUserBadgesCountResponse,
+  FETCH_USER_BADGES_COUNT,
+} from "../../../graphql/fetchUserBadgesCount";
+import {
+  UserProfileData,
+  FetchUserProfileDataResponse,
+  FETCH_USER_PROFILE_DATA,
+} from "../../../graphql/fetchUserProfile";
+
+import JoinedDateComponent from "./JoinedDateComponent";
+import { User } from "./ProfileHeaderComponent";
+
+export type BadgesDisplayedComponentProps = {
+  user: User;
+  earnedBadges: number;
+  totalBadges: number;
+};
+
+export default function UserProfileSection({
```

Comment:
Component name should correspond to what this component does

Reply:
changed to BadgesDisplayedComponent